src/imgui/ImGuiMachine.cc

```diff
@@ -279,6 +267,99 @@ void ImGuiMachine::showMenu(MSXMotherBoard* motherBoard)
 			});
 		}
 
+		setupSettingsOpen = im::Menu("Setup settings", true, [&]{
+			ImGui::TextUnformatted("When openMSX starts:");
+			ImGui::RadioButton("Load a machine", &startType, StartType::MACHINE);
+			HelpMarker("Select this option to load the specified machine when openMSX starts up, instead of a setup.");
+			im::Indent([&] {
+				im::Disabled(startType != StartType::MACHINE, [&] {
+					auto& allMachines = getAllMachines();
+					auto currentDefault = std::string(reactor.getDefaultMachineSetting().getString());
```

Comment:
Small tweak. I haven't tested, but I think it's fine to keep this as a `string_view` (avoid the `string` construction). And then only on line 282 converted this to a `string` (that should happen rarely).

Reply:
Fixed.